provisioning/miniprov/hare_mp/utils.py

```diff
@@ -185,7 +188,9 @@ def save_drives_info(self):
     def get_drive_info_from_consul(self, path: Text, machine_id: str) -> Disk:
         hostname = self.get_hostname(machine_id)
         disk_path = json.loads(str(path)).lstrip(os.sep)
-        drive_data = self.kv.kv_get(f'{hostname}/{disk_path}')
+        drive_data = None
+        while (not drive_data or drive_data is None):
+            drive_data = self.kv.kv_get(f'{hostname}/{disk_path}')
```

Comment:
Should we have a sleep of 2 to 5 seconds after each attempt?

Reply:
Good idea, may be waiting for 2 secs and continuing is good. Let me add that.